### ISSUE TITLE
save in `App.tsx` component is not triggering web browser page reload

### ISSUE THREAD

**Issue by uaandrei** [ogzhanolguncu/react-typescript-module-federation#6]
Hello, I'm trying to get module federation going by using your starter project, but I'm having issues because of changes not reflecting in web page. Can you please help?

### Issue description:
Web page is not reloading with changes after making modifications in `app1/src/App.tsx` component.

* before
```js
const App = () => (
  <Box margin="1.2rem">
    <Box>APP-1</Box>

    <Box>
      <CounterAppOne />
    </Box>
  </Box>
);

export default App;
```

* after
```js
const App = () => (
  <Box margin="1.2rem">
    <Box>APP-1 this change is not showing in browser after saving</Box>
    <Box>
      <CounterAppOne />
    </Box>
  </Box>
);

export default App;
```

* http://localhost:3001/ is not reloading. see screenshot 
![webpack-hmr-not-working-mf](https://user-images.githubusercontent.com/1614540/152575548-e1b4f21f-a0ec-4639-89c2-e692a08b1b2b.PNG)

* console logs:
```log
VM2379 log.js:26 [HMR] Update failed: ChunkLoadError: Loading hot update chunk main failed.
(missing: http://localhost:3001/main.c48a05b….hot-update.js)
    at http://localhost:3001/main.js:1265:26
    at new Promise (<anonymous>)
    at loadUpdateChunk (http://localhost:3001/main.js:1260:20)
    at http://localhost:3001/main.js:1720:29
    at Array.forEach (<anonymous>)
    at Object.__webpack_require__.hmrC.jsonp (http://localhost:3001/main.js:1715:22)
    at http://localhost:3001/main.js:862:47
    at Array.reduce (<anonymous>)
    at http://localhost:3001/main.js:858:55
log.js:24 [HMR] Updated modules:
log.js:24 [HMR]  - ./src/App.tsx
log.js:24 [HMR] App is up to date.
```

### Steps to reproduce:
1. Clone project
2. run `lerna bootstrap`
3. run `npm run start`
5. edit `app1\src\App.tsx`
6. save `app1\src\App.tsx`
7. http://localhost:3001/ not reloading to reflect changes
8. console has errors

_PS: Same issue happens when running from container_

**Comment by ogzhanolguncu** [comment]
I'll see what I can do.

**Comment by serhatgnc** [comment]
I have solved this problem on the `container` side of the project.
PR https://github.com/ogzhanolguncu/react-typescript-module-federation/pull/7

**Comment by ogzhanolguncu** [comment]
Thank you for the PR. Closing the issue
